### ISSUE TITLE
[Bug][A11Y][Filter Chip] No button role on filter chip

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#482]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

iPhone

### OS Version

Android and iOS

### App version

0.6.0

### Describe the issue

Filter chip has no role, we hear "selected, label, double-tap to unselect"

### Expected Behavior

Filter chip should have a **button role**

**Comment by pya35** [comment]
also, if possible: 
- move the hint "double tap to select/unselect" from the Label to the Hint property
- move the state selected/unselected in the value property

<img width="309" height="192" alt="" src="https://github.com/user-attachments/assets/af3c506b-243b-4cf1-88c0-64c6ade4fda6" />

**Comment by pya35** [comment]
OK
